cleanlab/classification.py

```diff
@@ -623,26 +705,62 @@ def find_label_issues(
 
         if self.verbose:
             print("Using predicted probabilities to identify label issues ...")
-        # Get the boolean mask of the label issues
         self.label_issues_mask = filter.find_label_issues(
             labels,
             pred_probs,
             **self.find_label_issues_kwargs,
         )
+        label_quality_scores = get_label_quality_scores(
+            labels, pred_probs, **self.label_quality_scores_kwargs
+        )
+        self.label_issues_df = pd.DataFrame(
+            {"is_label_issue": self.label_issues_mask, "label_quality": label_quality_scores}
+        )
+        if pred_probs is not None:
+            self.label_issues_df["given_label"] = labels
+            self.label_issues_df["predicted_label"] = pred_probs.argmax(axis=1)
```

Comment:
optional ---  if you don't want to store these is to just return a local version that has these two columns in it, without changing `self.label_issues_df`.

You don't lose any speed by doing this -- this function is not memoized at all and you recompute everything anyway when the function is called.

Reply:
dont fully grok this comment, clarify more?